### PR TITLE
Expand the output_scale comment

### PR DESCRIPTION
**Pull request by ba58smith**
There was a lack of understanding of the purpose of the `output_scale` member of the AnalogInput class. These additional comments will hopefully resolve that.

### CODE REVIEW

File: src/sensors/analog_input.h [hunk 1]
```diff
@@ -15,9 +15,18 @@ class AnalogInput : public NumericSensor {
...
    * the moment only ADC channel 1 (pins 32..39) is supported.
+   * 
    * @param[in] read_delay Time delay between consecutive readings, in ms
+   * 
    * @param[in] config_path Configuration path for the sensor
-   * @param[in] output_scale The maximum scale of the output
+   * 
+   * @param[in] output_scale The maximum scale of the output. The default is 1024,
+   * which is typical for Arduino AnalogInput pins. (Because of the way this has
+   * been implemented in SensESP, the default for ESP32 is also 1024.) If you want
```
Comment (joelkoz) [inline src/sensors/analog_input.h:25]: I think we should be explicit about "the way this has been implemented in SensESP".  Suggestion:

`@param output_scale The scale of the converted input value AnalogInput produces. The maximum raw analog value returned by an esp8266 and an esp23 are different, but AnalogInput takes that into account by converting them to a number between zero and output_scale. This allows the same code to run on both platforms...`

then continue with your examples

**Review by joelkoz — APPROVED** [review]
Left a suggested modification. Do with it what you think is best...

File: src/sensors/analog_input.h (hunk 1 above)
Comment (ba58smith) [reply src/sensors/analog_input.h:25]: Agreed. I made the change.

**Comment by mairas** [comment]
Merging.

**Pull request merged**